extensions/incubator/src/main/java/io/opentelemetry/extension/tracing/ExtendedTracer.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.extension.tracing;
```

Comment:
please please `io.opentelemetry.extension.trace` to be consistent with api 🗡️

Reply:
will do.

Reply:
wait. we can't do this, because we'll be putting the code in the same package as another module, which will possibly break users of the Java Module System.

Reply:
never mind, it's in the `extension` sub-package so it's all good.

Reply:
done